src/Tools/MicroBuild/run-pdbgit.ps1

```diff
@@ -0,0 +1,70 @@
+<#
+
+    Run PDB Git on our PDB files to enable proper source server support
+
+#>
+Param(
+    [string]$sourcePath = $null,
+    [string]$binariesPath = $null
+)
+
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+function Get-PackagesPath {
+    $packagesPath = $env:NUGET_PACKAGES
+    if ($packagesPath -eq $null) {
+        $packagesPath = join-path $env:UserProfile ".nuget\packages\"
+    }
+
+    return $packagesPath
+}
+
+try
+{
+    [xml]$deps = get-content (join-path $sourcePath "build\Targets\Dependencies.props")
+    $pdbGitVersion = $deps.Project.PropertyGroup.PdbGitVersion
+    $packagesPath = Get-PackagesPath
+    $pdbGit = join-path $packagesPath "pdbgit\$pdbGitVersion\tools\PdbGit.exe"
+
+    $signToolDataPath = join-path $sourcePath "build\config\SignToolData.json"
+    $signToolData = convertfrom-json (get-content $signToolDataPath -raw)
+    $allGood = $true
+    $count = 0
+
+    foreach ($relativePath in $signToolData.sign | %{ $_.values }) {
+        $binaryPath = join-Path $binariesPath $relativePath
+        $pdbPath = [IO.Path]::ChangeExtension($binaryPath, ".pdb")
+        if (-not (test-path $pdbPath)) {
+            continue;
+        }
+
+        write-host "Running pdbgit on $pdbPath"
+        & $pdbGit $pdbPath -u https://github.com/dotnet/roslyn --baseDir $sourcePath 
```

Comment:
What is this URL supposed to be? Is it the git repository (which I would expect should have a .git to be more canonical) or does it have some smarts here?

(I know GitHub currently supports clones without the .git, but unless you can get a contract with them saying they'll never break it...)